lib/java/test/org/apache/thrift/TestFullCamel.java

```diff
@@ -42,10 +41,11 @@ public void testCamelCaseSyntax() throws Exception {
     obj.setImFalse(true);
     byte[] serBytes = binarySerializer.serialize(obj);
     binaryDeserializer.deserialize(obj, serBytes);
-    assertTrue( obj.getABite() == (byte) 0xae );
-    assertTrue( obj.isImFalse() == true );
+    assertEquals(obj.getABite(), (byte) 0xae);
```

Comment:
Expected always comes before actual, in order for the output error message to display correctly:

```suggestion
    assertEquals((byte) 0xae, obj.getABite());
```